user/languages/ruby.md

```diff
@@ -259,19 +178,10 @@ HAML, Sinatra, Ruby on Rails,etc.
 To test against multiple versions of dependencies:
 
 1. Create a directory in your project's repository root where you will keep
-   gemfiles (./gemfiles is a commonly used name)
-2. Add one or more gemfiles to it
-3. Instruct Travis CI to use those gemfiles using the *gemfile* option in your
-   .travis.yml
-
-For example, amqp gem is [tested against EventMachine 0.12.x and 1.0
```

Comment:
While this is old-ish, seems the example to specify different Gemfiles, like gemfiles/eventmachine-pre linking to ruby-amqp, might still be relevant and helpful?